### PR TITLE
Add ray nomenclature of crustal phases

### PR DESCRIPTION
**Pull request by core-man**
Related to #7.

### CODE REVIEW

File: source/ray-theory-travel-times/ray-nomenclature/crustal-phases.rst [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+- **Sn**：在上地幔顶部折返的任何 S 波或上地幔顶部震源产生的上行 S 波
+
+- **Lg**：在较大的区域距离处，由整个地壳内部多次 S 波混响与 SV-P 和/或 P-SV 转换的叠加而产生的波群。
+  最大能量的群速度大约是 3.5 km/s。
+- **Rg**： 短周期地壳瑞利波
+
+.. figure:: crustal-phase-raypaths.png
+   :alt: 常见地壳震相的几何路径
+   :width: 60.0%
+   :align: center
```
Comment (seisman) [inline source/ray-theory-travel-times/ray-nomenclature/crustal-phases.rst]: Please optimize the image using https://squoosh.app

Comment (core-man) [reply source/ray-theory-travel-times/ray-nomenclature/crustal-phases.rst]: Use the defaults?

crustal-phase-raypaths

Comment (seisman) [reply source/ray-theory-travel-times/ray-nomenclature/crustal-phases.rst]: The default looks good for this image. You can drag the verticle line to left or right, to check the differences between the original image (left side) and the optimized image (right side).

Comment (core-man) [reply source/ray-theory-travel-times/ray-nomenclature/crustal-phases.rst]: How about the format? MozJPEG, browser JPEG or PNG?

Comment (seisman) [reply source/ray-theory-travel-times/ray-nomenclature/crustal-phases.rst]: They are different compression algorithms. Both JPEG and PNG are OK, although PNG is preferred.

File: source/ray-theory-travel-times/ray-nomenclature/crustal-phases.rst [hunk 2]
```diff
@@ -1,3 +1,35 @@
 地壳震相
 ========
```
Comment (seisman) [inline source/ray-theory-travel-times/ray-nomenclature/crustal-phases.rst:2]: Add a short introduction at the start?

Comment (core-man) [reply source/ray-theory-travel-times/ray-nomenclature/crustal-phases.rst:2]: ok

File: source/ray-theory-travel-times/ray-nomenclature/crustal-phases.rst [hunk 3]
```diff
@@ -1,3 +1,32 @@
...
+- **Lg**：在较大的区域距离处，由整个地壳内部多次 S 波混响与 SV-P 和/或 P-SV 转换的叠加而产生的波群。
+  最大能量的群速度大约是 3.5 km/s。
+- **Rg**： 短周期地壳瑞利波
+
+.. figure:: crustal-phase-raypaths.jpg
+   :alt: 常见地壳震相的几何路径
+   :width: 60.0%
+   :align: center
+
+   在局部和区域震中距（0°～20°）处，双层地壳模型中常见地壳震相的射线路径。
```
Comment (seisman) [inline source/ray-theory-travel-times/ray-nomenclature/crustal-phases.rst]:
```suggestion
   在局部和区域震中距（0°—20°）处，双层地壳模型中常见地壳震相的射线路径。
```

**Review by seisman — APPROVED** [review]

**Pull request merged**